benchmark/fs/bench-glob.js

```diff
@@ -25,10 +25,26 @@ async function main(config) {
   bench.start();
 
   for (let i = 0; i < config.n; i++) {
-    if (mode === 'async') {
-      noDead = await fs.promises.glob(pattern, { cwd: fullPath, recursive });
-    } else {
-      noDead = fs.globSync(pattern, { cwd: fullPath, recursive });
+    switch (mode) {
+      case 'sync':
+        noDead = fs.globSync(pattern, { cwd: fullPath, recursive });
+        break;
+      case 'promise':
+        noDead = await fs.promises.glob(pattern, { cwd: fullPath, recursive });
+        break;
+      case 'callback':
+        noDead = await new Promise((resolve, reject) => {
+          fs.glob(pattern, { cwd: fullPath, recursive }, (err, matches) => {
+            if (err) {
+              reject(err);
+            } else {
+              resolve(matches);
+            }
+          });
+        });
```

Comment:
Maybe add one version with util.promisify

Reply:
I don't really think so. It will make the benchmark run longer while giving no real benefit (we already measure the underlying callback-based function in this PR, and promisifying callback-based function makes little sense when promise-based function exists).